web/main.js

```diff
@@ -49,8 +49,7 @@ return {
   {
     // reset zoom
     var canvas = document.getElementById('canvas');
-    canvas.style.zoom = (window.innerWidth / window.outerWidth);
-    var width = window.outerWidth;
+    var width = window.innerWidth;
```

Comment:
In the process of testing I found a weird interaction with `window.outerWidth` seemingly being >100% on android firefox? So we'll just use `innerWidth`...

The goal is to use 100% of the visible viewport, erring on the side of <100% if there's any ambiguity. Note that we're still using `outerHeight` (not innerHeight) below, so there is room for more rigor here.